rest/data/admin.go

```diff
@@ -22,18 +24,239 @@ func (ac *DBAdminConnector) GetEvergreenSettings() (*evergreen.Settings, error)
 }
 
 // SetEvergreenSettings sets the admin settings document in the DB and event logs it
-func (ac *DBAdminConnector) SetEvergreenSettings(settings *evergreen.Settings, u *user.DBUser) error {
-	//TODO: remove these 3 after all the models have changed
-	if err := ac.SetAdminBanner(settings.Banner, u); err != nil {
-		return err
+func (ac *DBAdminConnector) SetEvergreenSettings(changes *restModel.APIAdminSettings,
```

Comment:
Can we add/document somewhere, what it takes to add a section to the config? 

It looks like we have to: 

- add the new structure,
- add add something here. 

other things? 

It seems like we could do some reflection magic here, not that I like that *too much*.

Reply:
I added this to config_registry.go. I couldn't get reflection to work with this method since the pointers didn't seem to get dereferenced correctly